index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="Stylesheet" href="./CSS/style.css"/>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Cynthia's portfolio</title>
+  </head>
+
+    <body>
+      <header class="main-header d-flex">
+        <h2>Welcome</h2>
+        <nav class="nav d-flex">
+          <ul class="nav-menu d-flex flex-center">
+            <li class="nav-link"><a href="#Portfolio">Portoflio</a></li>
+            <li class="nav-link active"><a href="#About">About</a></li>
+            <li class="nav-link"><a href="#Contact">Contact</a></li>
+          </ul>
+          <img src="./image/icons/Hamburger_icon.svg.png" class="show-menu-btn" alt="union">
```

Comment:
[OPTIONAL]: Please ensure that the hamburger button has a `nav` tag. You can use the hamburger icon, to get one, please check this [LINK](https://fontawesome.com/v5.15/icons?d=gallery&p=1&q=Hamburger). And if you would like to use an image as the hamburger, you should set it as CSS background-image instead of `img` tag according to the requirement.